### PR TITLE
redid import-image and fix ExportBin fixes #899

### LINKED ISSUE #899: There's a problem with exportBin
Screen Shot 2019-03-20 at 5 59 39 PM
cc @jywarren @HarshKhandeparkar @Divy123

### PR DESCRIPTION
**Pull request by tech4GT**
cc @jywarren

### CODE REVIEW

**Comment by tech4GT** [comment]
@jywarren I bumped the version, ready to merge here!

**Comment by tech4GT** [comment]
Also we should publish this to npm.

**Comment by jywarren** [comment]
oh oops it didn't pass...?

**Comment by tech4GT** [comment]
Let me close and re-open

**Comment by tech4GT** [comment]
Oh oops, let me have a look something is wrong!

**Comment by tech4GT** [comment]
For some reason the import-image in not working while running benchmarks, hmm I wonder why?

**Comment by tech4GT** [comment]
Screen Shot 2019-03-20 at 8 45 09 PM
Can't figure this out. This works normally but not while testing.
I have bypassed import-image in benchmarks until we can find a way for it to run.

**Comment by tech4GT** [comment]
cc @jywarren

**Comment by HarshKhandeparkar** [comment]
It says fs.readFileSync is not a function. Is nodejs installed properly. I used to get this error in webpack dev servers and it is a known one. Don't know why it is caused

**Comment by tech4GT** [comment]
@jywarren ready to merge now!

**Comment by tech4GT** [comment]
This seems to be a problem with urify so let's merge this and create a separate issue for that.

File: test/core/modules/benchmark.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ test('benchmark all modules', function(t) {
     if (mods.length > 1) { //Last one is test module, we need not benchmark it
       sequencer.steps[global.idx].output.src = image;
       global.idx++;
+      if (mods[0] === 'import-image') {
+        /* Not currently working */
+        console.log("Bypassing import-image");
```
Comment (jywarren) [inline test/core/modules/benchmark.js]: Oh, did you want to remove this first? Thanks!

**Comment by tech4GT** [comment]
Actually we need that to bypass import-image right now. Since I can't figure out how to get it to run while benchmarking, it always generates that error.
I was hoping we can fix that in a separate issue and leave it in for now?

**Comment by jywarren** [comment]
OK!

**Pull request merged**